### DISCUSSION TITLE
I want to validate input field dynamically  , how to do with formkit?

### DISCUSSION THREAD

**Discussion by yogendrarr** [formkit/formkit#1358] [General]
- If i am giving validation='required'  to given field so i want to be validate given field By using True False.
- If my condition is false then given input should not Validate it should optional .
- for example  in vee-validate. for ex->
- v-validate="{ required: (isEntityRoleMandatory == 1) }"
- can we do this in formkit validation?
-  by using this achieving validation on true false.
- <FormKit
        v-if="parseInt(accessDetailsResponse.isEntityNameVisible)"
        v-model="currentRow.entityName"
        :disabled="viewFlag"
        type="text"
        name="entityName"
        label="Entity Name"
        placeholder="enter entity name"
        validation="required"
       
        minlength="5"
        maxlength="50"
      /> how to do validation dynamically on this input field

**Comment by fenilli** [comment]
You can't as required does not accept a custom function, but you can create your own custom validation that does accept it.

**Reply by yogendrarr** [reply]
- i got solution for dynamically validate fields.
 
     :validation="[
         makeRequired
            ? ['required']
            : [],
        ]"

**Reply by fenilli** [reply]
Well that also works I guess haha
